Libplanet.Explorer.Executable/Program.cs

```diff
@@ -135,6 +116,37 @@ public static async Task Main(string[] args)
             }
         }
 
+        private static IStore LoadStore(Options options)
+        {
+            bool readOnlyMode = options.Seeds is null;
+            IStore store;
```

Comment:
Instead of reusing one variable for two purposes (one for decoration, and other one for inner implementation), how about introducing a new variable like `innerStore`?

Reply:
I love the suggestion, will apply it. Thanks. 🙏

Reply:
I applied the review and you can see the changes in 464d6694e05089b10e0c4205b96d2ca9c643b937. Please check it is the correct direction you thought.